roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -65,16 +75,10 @@ def run(self, tmp, task_vars):
 
     def required_images(self, task_vars):
         deployment_type = get_var(task_vars, "deployment_type")
-        # FIXME(juanvallejo): we should handle gracefully with a proper error
-        # message when given an unexpected value for `deployment_type`.
-        image_base_name = self.docker_image_base[deployment_type]
+        image_base_name = self.docker_image_base.get(deployment_type)
 
-        openshift_release = get_var(task_vars, "openshift_release")
-        # FIXME(juanvallejo): this variable is not required when the
-        # installation is non-containerized. The example inventories have it
-        # commented out. We should handle gracefully and with a proper error
-        # message when this variable is required and not set.
-        openshift_image_tag = get_var(task_vars, "openshift_image_tag")
+        openshift_release = get_var(task_vars, "openshift_release", default="latest")
+        openshift_image_tag = get_var(task_vars, "openshift_image_tag", default=openshift_release)
```

Comment:
@sosiouxme 
> openshift_release should be sanitized at this point to X.Y[.Z[.W]]. If the openshift_version role has run then it should be standardized to vX.Y.Z.W otherwise there is no guarantee it is even defined.

Since we are keeping `openshift_version` as a dep, do you think it is still worth having a check for a leading `v` in the value of `openshift_image_tag`?